main.js

```diff
@@ -0,0 +1,149 @@
+        topic +
+        "cercle&type=video&videoEmbeddable=true&key=AIzaSyCjo4u-wcr_ExFNPxiYlWZP3LLr-ythijE"
+      );
+    }
+  },
+
+  model: VideoModel,
+
+  initialize: function () {
+    this.on("add", function (model) {
+      model.fetch();
+    });
+  },
+  
+  parse: function (response) {
+    return response.items.map(function (x) {
+      return {
+        videoId: x.id.videoId,
+        videoThumbnail: x.snippet.thumbnails.default.url,
+        videoTitle: x.snippet.title,
+        videoDesc: x.snippet.description,
+      }
+    })
+  }
+});
+
+var AppModel = Backbone.Model.extend({
+  defaults: function () {
+    return {
+      videos: new VideosCollection(),
+    };
+  }
+});
+
+var VideoView = Backbone.View.extend({
+  className: 'video',  // to reference later
+
+  template: Handlebars.compile($('#five-display-template').html()),
+
+  render: function () {
+    this.$el.html(this.template(this.model.toJSON()));
+
+    return this;
+  },
+});
+
+var MainVideoView = Backbone.View.extend({
+  className: 'mainVideo',
+
+  template: Handlebars.compile($('#main-display-template').html()),
+
+  render: function () {
+    this.$el.html(this.template(this.model.toJSON()));
+
+    return this;
+  }
+})
+
+var AppView = Backbone.View.extend({
+  el: $("body"),
+
+  events: {
+    "click .search": "newSearch", 
+    "click .click-here": "newMain"
+  },
+  
+  initialize: function () {
+    this.$search = this.$('#search');
+    this.listenTo(this.model.get('videos'), 'change', this.renderVideo);
+    this.listenTo(this.model.get("videos"), "change", this.renderMainVideo);
+
+    this.$mainspot = this.$('.five-spot');
+    this.listenTo(this.model.get('videos'), 'reset', this.renderVideos);
+    this.listenTo(this.model.get("videos"), "reset", this.renderMainVideo);
+    
+    this.renderVideos;
+  },
+
+  newSearch: function () {
+    var inputTopic = this.$('.topic').val();
+
+    this.model.set({topic: inputTopic});
+    // listen to change of model should notice new url and run render functions
+  },
+
+  renderVideo: function (model) {
+    var videoView = new VideoView({ model: model });
+
+    this.$('.five-display').append(videoView.render().el);
+  },
+
+  renderVideos: function () {
+    this.model.get('videos').each(function (v) {
+      this.renderVideo(v);
+    }, this);
+  },
+
+  renderMainVideo: function (model, vidId) {
+    // if no videos have been clicked, take the first
+    if (!vidId.length) {
+      var mainVideoView = new MainVideoView({ model: model.at(0) });
+    } else {
+      var whereItsAt = this.model.get('videos').findWhere({ videoId: vidId });
+      
+      if (mainVideoView) {
+        this.mainVideoView.remove();
+        console.log('here');
+      }
+
+      var mainVideoView = new MainVideoView({ 
+        model: whereItsAt });
+    }
+
+    this.$(".main-display").append(mainVideoView.render().el);
+  },
+
+  newMain: function (e) {
+    newId = $(e.currentTarget).data().id;
```

Comment:
Never leave out `var` - this will create an immediate global variable which can be devastating.

Also, start your function name with a verb.